src/sentry/services/hybrid_cloud/replica/impl.py

```diff
@@ -76,7 +76,7 @@ def get_conflicting_unique_columns(
     scope = category.get_scope()
     scope_controlled_columns: List[str]
     if scope == scope.USER_SCOPE:
-        scope_controlled_columns = [get_foreign_key_column(destination, User)]
+        scope_controlled_columns = ["ident", get_foreign_key_column(destination, User)]
```

Comment:
Is it worth leaving a comment here about this being for `AuthIdentity`? I would see this causing problems in the future if we have another user scope replicated model.

Reply:
You are correct!  I'll add a comment on my next pass.